Lib/test/test_complex.py

```diff
@@ -241,6 +241,13 @@ def test_pow(self):
         else:
             self.fail("should fail 0.0 to negative or complex power")
 
+        try:
+            0j ** (-1)
+        except ZeroDivisionError:
+            pass
+        else:
+            self.fail("should fail 0.0 to negative or complex power")
```

Comment:
Could you revert this change? Since the test file is originated from CPython project, we usually don't edit it ourselves.